ch04.adoc

```diff
@@ -185,97 +187,317 @@ The `computed_standard_name` attribute indicates that the values in variable
+Because the calendars have different sets of valid dates, and different treatments of leap seconds (see below in this section, and <<leap-seconds>>), a given time coordinate value with given **`units`** can represent different datetimes in different calendars; conversely, a given datetime is represented by different time coordinate values in different calendars.
+Moreover, in different calendars a given datetime can identify a different instant in the continuous physical dimension of time.
+// Examples of these situations are shown in the table of *Appendix ???*.
 
-* Any date/times between the end of the 60th second of the last minute of one hour and the start of the first second of the next hour cannot be represented by time coordinates e.g. **`2016-12-31 23:59:60.5`** cannot be represented.
-* A time coordinate value must not be interpreted as representing a date/time in the excluded range.
-For instance, **`60 seconds after 23:59`** means **`00:00`** on the next day.
-* A date/time in the excluded range must not be used as a reference date/time e.g. **`seconds since 2016-12-31 23:59:60`** is not a permitted value for **`units`**.
-* It is important to realise that a time coordinate value does not necessarily exactly equal the actual length of the interval of time between the reference date/time and the date/time it represents.
+The lengths of the months in the Gregorian calendar are used in all calendars except **`360_day`**, **`none`** (see <<none-calendar>>) and explicitly defined calendars (see <<explicit-calendar>>).
+The calendars differ in their treatment of leap years (when there are 29 days in February instead of 28).
 
-The values currently defined for **`calendar`** are listed below.
-In all calendars except **`360_day`** and **`none`**, the lengths of the months are the same as in the Gregorian calendar for leap years and non-leap years.
-In the **`julian`** and the default **`standard`** mixed Gregorian/Julian calendar, dates in years before year 0 (i.e. before 0-1-1 0:0:0) are not allowed, and the year in the reference date/time of the units must not be negative.
+Leap seconds are adjustments made at irregular and unpredictable intervals in Coordinated Universal Time (UTC), the international basis of civil timekeeping in the real world.
+In response to slight variations in the Earth's rotation speed, positive or negative leap seconds are inserted in order to keep UTC close to mean solar time at 0 `degrees_east` i.e. the time zone with the default (zero) time zone offset in UDUNITS and CF (see <<time-coordinate-units>>).
+When a positive leap second is introduced at the end of a minute, that minute contains 61 seconds; 27 leap seconds have been added to UTC since 1958, most recently at the end of 2016.
+A minute would contain 59 seconds if it included a negative leap second, but none have been introduced up to now.
+It has been decided that after 2035 no further leap seconds will be added.
```

Comment:
decided by whom? ISO ?

probably not critical to add that, but it's as simple as:

"It has been decided by ISO, that..."

maybe worth it.

Reply:
I've deleted that bit.